extensions/google-shopping-ads/release-notes/index.md

```diff
@@ -15,6 +15,30 @@ The release notes include:
 
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
+-   {:.bug}Known issues
+
+### v2.0.0
+
+Google Shopping ads Channel 2.0.0 is generally available for versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.fix} **Resolved Configurable Product Issues**: Configurable product (parent/child) handling has been improved.
+    - <!--3372-->Corrected an issue preventing child products from inheriting the parent description and causing Google to disapprove products for missing descriptions
+    - <!--3297, 3341, 3410-->Corrected and improved the gtag (pageview) trigger when selecting options for a child product and when viewing a child product in the cart
+
+- {:.fix} **Index Management Improvements**: Indexing processes between Magento and Google have been updated for increased efficiency and performance.
+    <!--3004, 3298, 3343, 3480, 3481, 3465, 3472-->
+    - Consolidated and rebuilt indexer for communicating with Google
+    - Set indexer to run on schedule by default
+    - Adjusted indexer logic to only send product data changes, instead of all data for a changed product
+
+- {:.fix} **Security Updates**: <!--3380-->Updated security whitelist and implemented Content Security Policy (CSP) when communicating with Google.
+
+- {:.fix} **Update to Product Linking**: <!--3463-->Updated communication between Google and Magento to ensure use of the same product link to prevent product disapprovals in the Google Merchant Account (GMC).
+
+- {:.fix} **Product Attribute Updates**: <!--3485-->Updated attribute management to allow the deletion of the Google Merchant Center Category attribute, which can be created by the Google application.
+
+- {:.new} **Magento Log Exceptions**: <!--3439-->Implemented **Enable Services Logs** feature to help in troubleshooting. See the [Channels](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) topic in the Configuration Reference.
```

Comment:
Link doesn't work. 404. It's not in prod 2.3:
![image](https://user-images.githubusercontent.com/29388020/60192147-07812380-97fb-11e9-9f0b-923fbdf63f4c.png)

Reply:
Removing this feedback. Looks like 2.0 merchdocs PR not merged yet. 👍